src/main/scala/where/Condition.scala

```diff
@@ -76,3 +78,15 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class HasNoRelationships(node: Node, labels: Seq[Label] = Seq.empty) extends Condition {
+  def toQuery(identifiableMap: IdentifiableMap): String = {
+    val identifier = identifiableMap.get(node) getOrElse (throw new IdentifierDoesntExistException())
+    val labelsQuery = labels map (_.toQuery) mkString "|"
```

Comment:
we should reuse the logic from `Relationship` to generate this (note that it needs a ":" at the front). what i would do is move `kindsToQuery` and `escape` out of the `Relationship` trait and into a companion object, so then it's available here. the fact that `kinds` in relationships are `String` appears to be a mistake, they should be `Label`... you can try and make that change but if you have trouble with the implicits tomorrow let me know (since strings automatically convert to labels)